parser.go

```diff
@@ -221,3 +221,9 @@ func isSpace(r rune) bool {
 	}
 	return false
 }
+
+
+// isNewLine reports whether the rune is a new line character
+func isNewLine(r rune) bool {
+	return r == '\n'
```

Comment:
should also detect `\r` so CR LF on windows will detect the first one, otherwise we will cut after CR

Reply:
Fixed by just trimming the end on any kind of space char when unquoted.

Reply:
ok, make sense